contracts/wrapperContracts/WrapFeeBurnerWrapper.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.4.18;
+
+
+import "../Withdrawable.sol";
+
+
+interface WrapFeeBurner{
```

Comment:
for my understanding - what is the purpose of declaring this interface? (and interfaces in general)